### PR TITLE
Fixes #23987: Add policy mode info to node compliance API

### PR DESCRIPTION
**Pull request by clarktsiory**
https://issues.rudder.io/issues/23987

### CODE REVIEW

File: webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/ComplianceApi.scala [hunk 1]
```diff
@@ -412,12 +416,13 @@ class ComplianceApi(
  * compliance for all rules/nodes/directives.
  */
 class ComplianceAPIService(
-    rulesRepo:                   RoRuleRepository,
-    nodeInfoService:             NodeInfoService,
-    nodeGroupRepo:               RoNodeGroupRepository,
-    reportingService:            ReportingService,
-    directiveRepo:               RoDirectiveRepository,
-    val getGlobalComplianceMode: () => Box[GlobalComplianceMode]
```
Comment (fanf) [inline webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/ComplianceApi.scala:420]: are you sure it's not used in a plugin or something? It's strange enough to have a val here to be cautious

Comment (fanf) [reply webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/ComplianceApi.scala:420]: ok, I didn't find anything :shrug:

**Review by fanf — APPROVED** [review]
LGTM

**Comment by fanf** [comment]
OK, merging this PR

File: webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/ComplianceApi.scala (hunk 1 above)
Comment (clarktsiory) [reply webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/ComplianceApi.scala:420]: I also didn't find anything :smile_cat:

**Pull request merged**